test-cases/tests/multiAZ/p01-measure-downtime-during-down-az.md

```diff
@@ -44,13 +45,16 @@ Measure the downtime of the RHOAM components during a AWS Availability Zone fail
    export GRAFANA_DASHBOARD=true
    export RHOAM=true
    make local/deploy
+   oc project workload-web-app
+   oc scale dc workload-web-app --replicas=3
```

Comment:
One liner can be used for this (inspired by Laura): `oc scale dc workload-web-app --replicas=3 -n workload-web-app`

Reply:
Also, quite minor (optional)

Reply:
:+1: